addons/l10n_tr_nilvera_edispatch/views/stock_picking_views.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_picking_internal_search_inherit_l10n_tr_nilvera_edispatch" model="ir.ui.view">
+        <field name="name">stock.picking.internal.search.inherit.l10n.tr.nilvera.edispatch</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_internal_search"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group" position="inside">
+                <filter 
+                    string="GİB e-Dispatch State"
+                    name="l10n_tr_nilvera_dispatch_state"
+                    context="{'group_by': 'l10n_tr_nilvera_dispatch_state'}"
+                />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_picking_form_inherit_l10n_tr_nilvera_edispatch" model="ir.ui.view">
+        <field name="name">view.picking.form.inherit.l10n.tr.nilvera.edispatch</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//header" position="inside">
+                <button name="action_generate_l10n_tr_edispatch_xml" type="object" invisible="state != 'done' or country_code != 'TR'" string="Generate GİB e-Dispatch (XML)" />
```

Comment:
Can you add the != outgoing too here?

Currently, the button is visible for incoming too, while it is not supposed to